Lib/webbrowser.py

```diff
@@ -19,6 +19,31 @@ class Error(Exception):
 _tryorder = None                # Preference order of available browsers
 _os_preferred_browser = None    # The preferred browser
 
+_browsers_types = [
```

Comment:
This list includes obsolete browsers removed by #102872.

Reply:
i also dislike that type of hard coded stuff.
but i also dont know a better solution here.
anyway that module has a lot of hard code stuff inside.

Reply:
These are the browsers that were removed in 3.12:
"galeon",
"skipstone",
"mosaic",
"grail",

Could you please remove these from the list?

Reply:
For this PR, the change would be to remove the list.
If another ticket is created to add `--list` option to webbrowser, there the correct list should be added, possibly built dynamically by the `register` function to avoid this kind of issue!

Reply:
https://github.com/kr-g/cpython/commit/308e060f0c67510c0af0f39b3418bab756c7e6ce

Reply:
read at the end. do send proposals. i will accept any.

Reply:
i think this one needs rework.
i checked against the procedure how `register_standard_browsers()` works and got a resulting list with some strange entries. but it seams that those are just aliases for others. so i will remove this list completly